### PR TITLE
Feat/ Edit style WizardForm, TextRow and TaskList

### PR DESCRIPTION
**Pull request by dottyy**
This PR introduces style enhancements for the following components:

WizardForm:

Improved the visual appearance for better user guidance.

TextRow:

Refined typography and spacing for improved readability.
Ensured alignment with the overall design system.

TaskList:

Updated styles for better task organization and clarity.
Enhanced the user interface with new color schemes and layout adjustments.

### CODE REVIEW

File: src/components/extensive/WizardForm/FormHeader.tsx [hunk 1]
```diff
@@ -17,28 +17,30 @@ export const WizardFormHeader = (props: WizardFormHeaderProps) => {
...
-              <Case condition={props.errorMessage}>{t("Something went wrong.")}</Case>
-            </Switch>
-          </Text>
+          <div className="flex">
+            <Text variant="text-14-light" className="overflow-hidden text-ellipsis whitespace-nowrap">
+              <T _str="Progress: {number} steps complete" number={`${props.currentStep}/${props.numberOfSteps}`} />
+            </Text>
+            <Text variant="text-14-light">
+              <Switch>
+                <Case condition={props.formStatus === undefined}>{t("Unsaved")}</Case>
```
Comment (pachonjcl) [inline src/components/extensive/WizardForm/FormHeader.tsx:34]: I think this condition will be better use as `condition={!props.formStatus}`

File: src/components/extensive/WizardForm/index.tsx [hunk 2]
```diff
@@ -105,8 +107,6 @@ function WizardForm(props: WizardFormProps) {
       }
       props.onChange && props.onChange(formHook.getValues());
       props.onStepChange?.(data, selectedStep);
-      formHook.clearErrors();
-      formHook.trigger();
```
Comment (pachonjcl) [inline src/components/extensive/WizardForm/index.tsx:109]: why are you deleting those?

File: src/components/extensive/WizardForm/index.tsx [hunk 3]
```diff
@@ -134,8 +134,6 @@ function WizardForm(props: WizardFormProps) {
         });
       });
     }
-    formHook.reset(formHook.getValues());
-    formHook.trigger();
```
Comment (pachonjcl) [inline src/components/extensive/WizardForm/index.tsx:138]: again, why are you deleting those?

File: src/components/extensive/WizardForm/index.tsx [hunk 4]
```diff
@@ -162,45 +160,64 @@ function WizardForm(props: WizardFormProps) {
...
-              selectedStepIndex < lastIndex
-                ? props.nextButtonText || t("Save and continue")
-                : props.submitButtonText || t("Submit"),
-            onClick: formHook.handleSubmit(onSubmitStep),
-            disabled: (selectedStepIndex === lastIndex && props.submitButtonDisable) || formHasError
-          }}
-        />
-      </FormStep>
+                : undefined
+            }
```
Comment (pachonjcl) [inline src/components/extensive/WizardForm/index.tsx:209]: if you are going to be using a conditional in the following way:
```
property={someCondition ? component : undefined}
```

there is no point in the undefined, please rewrite it as:
```
property={someCondition && component}
```

**Review by pachonjcl — CHANGES_REQUESTED** [review]

File: src/components/extensive/WizardForm/FormHeader.tsx (hunk 1 above)
Comment (dottyy) [reply src/components/extensive/WizardForm/FormHeader.tsx]: done

File: src/components/extensive/WizardForm/index.tsx (hunk 2 above)
Comment (dottyy) [reply src/components/extensive/WizardForm/index.tsx]: remove code

File: src/components/extensive/WizardForm/index.tsx (hunk 3 above)
Comment (dottyy) [reply src/components/extensive/WizardForm/index.tsx]: remove code

File: src/components/extensive/WizardForm/index.tsx (hunk 4 above)
Comment (dottyy) [reply src/components/extensive/WizardForm/index.tsx:213]: sorry, but backButtonProps acept undefined and Type 'false' has no properties in common with type 'Omit<IButtonProps, "color" | "variant">'.

**Review by pachonjcl — APPROVED** [review]

File: src/components/extensive/WizardForm/index.tsx [hunk 5]
```diff
@@ -162,45 +164,64 @@ function WizardForm(props: WizardFormProps) {
...
+            <div className="flex gap-4 rounded-lg bg-tertiary-80 p-6">
+              <Text variant="text-16-bold" className="text-white">
+                {t(`Note: Project polygons are editable through a new geometry-focused workflow that is accessible by
+                clicking on this link.`)}
+              </Text>
+              <Button
+                variant="text"
+                className="text-14-bold nowrap whitespace-nowrap rounded-lg border-2 border-transparent bg-[#ffb88891] px-4 py-0 uppercase text-white hover:border-white"
+              >
+                edit polygon
```
Comment (roguenet) [inline src/components/extensive/WizardForm/index.tsx:185]: Needs a wrap in `t`

File: src/components/extensive/WizardForm/index.tsx [hunk 6]
```diff
@@ -162,45 +164,64 @@ function WizardForm(props: WizardFormProps) {
...
-          }}
-        />
-      </FormStep>
+                : undefined
+            }
+            submitButtonProps={{
+              children:
+                selectedStepIndex < lastIndex
+                  ? props.nextButtonText || t("Save and continue")
+                  : props.submitButtonText || t("Submit"),
```
Comment (roguenet) [inline src/components/extensive/WizardForm/index.tsx:218]: `??` on both lines

File: src/components/extensive/WizardForm/index.tsx (hunk 5 above)
Comment (dottyy) [reply src/components/extensive/WizardForm/index.tsx]: Thanks for the feedback! I've wrapped the text in t as suggested.

File: src/components/extensive/WizardForm/index.tsx (hunk 6 above)
Comment (dottyy) [reply src/components/extensive/WizardForm/index.tsx]: Thanks for the suggestion! I've updated the code to use the nullish coalescing operator (??) on both lines as recommended:


`children:
  selectedStepIndex < lastIndex
    ? props.nextButtonText ?? t("Save and continue")
    : props.submitButtonText ?? t("Submit"),`

**Review by roguenet — APPROVED** [review]

**Pull request merged**